presto-native-execution/presto_cpp/presto_protocol/core/presto_protocol-json-hpp.mustache

```diff
@@ -246,7 +246,7 @@ namespace facebook::presto::protocol {
         {{/fields}}
         {{#comparable}}
             virtual bool operator<(const {{&class_name}}& /* o */) const {
-                throw std::runtime_error("missing operator<() in {class_name} subclass");
+                throw std::runtime_error("missing operator<() in {{class_name}} subclass");
```

Comment:
This was causing the error message to display as `missing operator<() in {class_name} subclass`. The double braces are needed to correct it when generating the protocol.